### DISCUSSION TITLE
Question About Possible Header Inconsistency

### DISCUSSION THREAD

**Discussion by ghost** [colinwilson/lotusdocs#19] [General]
The theme's [homepage](https://lotusdocs.dev/) has Docs, Community, and Social icons in the header. When navigating to Docs, the header shows Social icons and a theme adjustment icon, but not Community. 

Is this a bug?

**Comment by colinwilson** [comment]
Ah, not really a bug.

The 'Docs' and 'Community' menus on the landing page are currently controlled manually via the `[[menu.primary]]` parameter in `hugo.toml`. I haven't yet documented this on [lotusdocs.dev/docs/](https://lotusdocs.dev/docs/)

```
[menu]
[[menu.primary]]
    name  = "Docs"
    url = "docs/"
    identifier = "docs"
    weight = 10

[[menu.primary]]
    name  = "Community"
    url = "https://github.com/colinwilson/lotusdocs/discussions"
    identifier = "community"
    weight = 30
```
I considered adding a similar menu to the `/docs` section but the mobile view started to get cluttered. Will have another look into how this could be implemented (cleanly) though as it's definitely a nice option to have.
